src/pages/public_registration/LogPanel.tsx

```diff
@@ -19,6 +22,8 @@ interface LogPanelProps {
   registration: Registration;
 }
 
+const tooltipDelay = 400;
```

Comment:
noen spesiell grunn til at vi ønsker en liten delay her?

Reply:
Uten delay så kommer den opp med en gang, noe som er unaturlig. Det er gjort extensive research på hva optimal delay for tooltip er, og det er 0,3-0,5ms. Så jeg tok det som var midt i mellom der :)

Reply:
ah, aldri tenkt på. alltid tenkt at raskest er best :D